<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Max Total Steps:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="max_total_steps"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="free_simulation">
          <property name="text">
           <string>Free simulation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirm_button">
          <property name="text">
           <string>confirm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
